theme.json

```diff
@@ -931,6 +931,11 @@
 			"lineHeight": "1.4"
 		},
 		"blocks": {
+			"core/avatar": {
+				"border": {
+					"radius": "100px"
```

Comment:
Would it make sense to create some radius presets, of which one preset could be "round", and just set to 9999px?

Reply:
There is a request in the Gutenberg repository to add this to the editor.
Let's check how far that has come, but I think it needs dev:
https://github.com/WordPress/gutenberg/issues/64041

Reply:
I will add the issue to the list of issues.